index.md

```diff
@@ -0,0 +1,98 @@
+# Workshop Wise Index
+
+## Table of Contents
+
+1. [Golang Workshop](#golang-workshop)
+1. [Google Summer of Code Introductory Seminar](#gsoc-introductory-seminar)
+1. [Git and Github Workshop](#git-and-github-workshop)
+1. [KOSS Lightning Talks](#koss-lightning-talks)
+1. [KOSS Freshers' Introductory Seminar](#koss-freshers-introductory-seminar)
+1. [Linux Install Fest](#linux-installation-fest)
+1. [Miscellaneous](#miscellaneous)
+    1. [Linux Shell Scripting and Vim Commands](#linux-shell-scripting-and-vim-commands)
```

Comment:
I think just the top level miscellaneous link is enough.

Reply:
probably yeah, but no harm in listing them out. For example I didn't even know koss conducted a hacking workshop before I made this index.

Reply:
Okay